WalletWasabi/Mono/Option.cs

```diff
@@ -183,7 +183,7 @@ protected Option(string prototype, string description, int maxValueCount, bool h
 
 			if (prototype.Length == 0)
 			{
-				throw new ArgumentException("Cannot be the empty string.", nameof(prototype));
+				throw new ArgumentException("Cannot be empty string.", nameof(prototype));
```

Comment:
I think a better alternative is "Cannot be an empty string."

Reply:
I agree.